### PR TITLE
reduce test verbosity, replace a deprecated method with a java native implementation

### PR DESCRIPTION
**Pull request by MidNight-er**

### CODE REVIEW

**Comment by MidNight-er** [comment]
refactoring

**Comment by MidNight-er** [comment]
gave up to tackle with the check style )

File: apache-jsp/src/test/java/org/eclipse/jetty/jsp/TestJettyJspServlet.java [hunk 1]
```diff
@@ -56,23 +58,23 @@ public static class DfltServlet extends HttpServlet
...
+            resp.getOutputStream().println(DEFAULT_RESP_OUTPUT);
         }
     }
 
     @BeforeEach
     public void setUp() throws Exception
     {
         JspFactory.setDefaultFactory(new JspFactoryImpl());
-        File baseDir = MavenTestingUtils.getTestResourceDir("base");
+        final Path path = Path.of("src", "java", "resources", "base").toAbsolutePath();
```
Comment (joakime) [inline apache-jsp/src/test/java/org/eclipse/jetty/jsp/TestJettyJspServlet.java:69]: Do not change this to normal Path behaviors.
When we merge up, and the build/test gets parallelized the technique you are using will fail.
It has to use the `${project.base}` system property (when present) to find the directory correctly.

Use `Path baseDir = org.eclipse.jetty.toolchain.test.MavenPaths.findTestResourceDir("base");` instead if you want to get rid of the deprecation warning.

**Comment by joakime** [comment]
> gave up to tackle with the check style )

Looks like your import statement order was wrong.
If you leave the order alone you should have been ok.

Tip: build locally, via command line, `mvn clean install -DskipTests` to quickly build and have checkstyle do it's thing.
Also, if you use an IDE, the checkstyle rules are checked into the project at `build-resources/src/main/resources/jetty-checkstyle.xml` so you can get the rules told to you in an interactive way as you edit the code.

**Pull request closed without merging**